### PR TITLE
Add Support Notice

### PR DESCRIPTION
**Pull request by jasonpatrickellykrause**
Adding a line to the readme to let users know that installation can prevent them from receiving support.

### CODE REVIEW

**Comment by RandomNinjaAtk** [comment]
Hmm, thanks. I agree that something like this is needed. However, I feel like it implies the scripts modify Lidarr itself. Which is not the case....

The Lidarr support staff even thinks it modifies Lidarr, which is just untrue. The scripts don't modify Lidarr itself in any manor that a user wouldn't already be able to modify themself using the web-ui.

However at the same time, I'm not sure if there is a better way to word it... Because I don't want users to have the impression that this alters the program at all (as in tamper), when it indeed does not do that.

File: lidarr/readme.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # README
 
+## Notice
+
+Please note that use of arr-scripts modified Lidarr's behavior and is not supported by the Lidarr community.
```
Comment (RandomNinjaAtk) [inline lidarr/readme.md]: Maybe change it to:

"Please note that use of arr-scripts is not supported by the Lidarr community.

File: lidarr/readme.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Notice
 
-Please note that use of arr-scripts modified Lidarr's behavior and is not supported by the Lidarr community.
+Please note that the use of arr-scripts modifies Lidarr's behavior and is not supported by the Lidarr community.
```
Comment (RandomNinjaAtk) [inline lidarr/readme.md:5]: Maybe change it to:

"Please note that use of arr-scripts is not supported by the Lidarr community."

**Comment by RandomNinjaAtk** [comment]
Could you also add these disclaimers to the other arr-script apps. (radarr/sonarr/readarr/sabnzbd)

Thanks!

File: lidarr/readme.md (hunk 2 above)
Comment (jasonpatrickellykrause) [reply lidarr/readme.md:5]: This is perfect.

**Comment by RandomNinjaAtk** [comment]
Resolved over the last few commits: https://github.com/RandomNinjaAtk/arr-scripts/commit/25b7947a05a5a5564c243f688e619031d9abd385

**Pull request closed without merging**